src/settings/ExportFundSettings/ExportFundSettings.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import moment from 'moment';
+
+import {
+  Pane,
+  Button,
+  Select,
+  Label,
+  Loading,
+  Row,
+  Col,
+} from '@folio/stripes/components';
+import { exportCsv } from '@folio/stripes/util';
+import { useShowCallout } from '@folio/stripes-acq-components';
+
+import { useFiscalYearOptions } from './useFiscalYearOptions';
+import { useExportFund } from './useExportFund';
+import css from './ExportFundSettings.css';
+import { EXPORT_FUND_FIELDS } from './constants';
+
+const ExportFundSettings = () => {
+  const showCallout = useShowCallout();
+  const [fiscalYearId, setFiscalYearId] = useState();
+  const { fiscalYearOptions, isLoading: isFYLoading } = useFiscalYearOptions();
+
+  const { fetchExportFund } = useExportFund(fiscalYearId);
+
+  const onChangeFY = (e) => {
+    setFiscalYearId(e.target.value);
+  };
+
+  const onExportFund = async () => {
+    const { data: exportFund, isError } = await fetchExportFund();
+
+    if (isError) {
+      return showCallout({
+        messageId: 'ui-finance.fund.actions.load.error',
+        type: 'error',
+      });
+    }
+
+    const filename = `fund-codes-export-${moment().format('YYYY-MM-DD-hh:mm')}`;
+
+    await exportCsv(
```

Comment:
this with filename generation can be moved in utils + let's include FY in file name to give more context